README.md

```diff
@@ -22,14 +22,23 @@ Or install it yourself as:
 
 ## Usage
 
-TODO: Write usage instructions here
+The following config attributes need to be set when using the gem:
+
+```ruby
+Marqeta.configure do |config|
+  config.logger = <CUSTOM_LOGGER>
+  config.username = <MARQETA API USERNAME>
+  config.password = <MARQETA API PASSWORD>
+  config.base_url = <MARQETA API BASE URL>
+end
+```
 
 ## Development
 
-After checking out the repo, run `bin/setup` to install dependencies. Then, run `rake test` to run the tests. You can also run `bin/console` for an interactive prompt that will allow you to experiment.
+After checking out the repo, run `bundle` to install dependencies. Then, run `rspec` to run the tests. You can also run `bin/console` for an interactive prompt that will allow you to experiment.
```

Comment:
Minor, but `bin/setup` already does this. I think we should follow the convention.

https://github.com/github/scripts-to-rule-them-all
https://robots.thoughtbot.com/bin-setup